test/benchmark/download-owasp-v2-rules.sh

```diff
@@ -2,10 +2,7 @@
 #
```

Comment:
I think we must remove this file. CRS2 is unmaintained, I'm sure nobody uses that.

And from the other hand: the seclang parser fails to parse it :(
```
phase:4,rev:'2',ver:'OWASP_CRS/2.2.9',maturity:'9',accuracy:'9',chain,t:none,capture,ctl:auditLogParts=+E,block,msg:'ASP/JSPProblems loading the rules...
Rules error. File: owasp-v2/base_rules/modsecurity_crs_50_outbound.conf. Line: 39. Column: 387. Expecting an action, got:  source code leakage',logdata:'Matched Data: %{TX.0} found within %{MATCHED_VAR_NAME}: %{MATCHED_VAR}',id:'970903',tag:'OWASP_CRS/LEAKAGE/SOURCE_CODE_ASP_JSP',tag:'WASCTC/WASC-13',tag:'OWASP_TOP_10/A6',tag:'PCI/6.5.6',severity:'3'"
```

so just remove this.

Reply:
That's alright, will do (and rebase the PR too).